### ISSUE TITLE
Fatigue Indicator Icon: not relieving on round out

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#304]
```
Before posting, please read the following guidelines for posts in the issue
tracker:
http://code.google.com/p/andors-trail/wiki/Forums_vs_issuetracker


What steps will reproduce the problem?
1. Go up to BWM
2. Fight some Wyrn [Assistant, Trainers] 
3. Notice rounds [check in status until 1 left] then return to gameplay and 
notice Fatigue doesn't relieve.

What is the expected output? What do you see instead?
Expected.. Icon dropping away. 
Result: Icon just stuck more or less [until one manually checks status again]

What version of the product are you using? On what device?
Started occurring 0.6.11a and continues in b1

Please provide any additional information below.


```

Original issue reported on code.google.com by `insin...@gmail.com` on 30 May 2012 at 5:13

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report, great find!

This should now be fixed in r252.
```

Original comment by `oskar.wi...@gmail.com` on 30 May 2012 at 6:59
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
